### ISSUE TITLE
White Bar At Top Of Frameless Window After Maximize Cycle

### ISSUE THREAD

**Issue by CoreAutomation-JD** [electron/electron#13193]
* Electron Version: 2.0.2
* Operating System (Platform and Version): Windows 10 (17134)
* Last known working Electron version: N/A

**Expected Behavior**
When maximizing a frameless window, then unmaximizing and then maximizing again, there should not be a white bar atop the window. 

**Actual behavior**
The first maximize works. Removing maximize also works. Immediately maximizing again (without resizing the window) leaves a white bar several pixels high atop the browserwindow.

**To Reproduce**
Use the latest Discord app (or various others like Spotify), follow the workflow above to reproduce this very easily.

Further discussion here: https://www.reddit.com/r/Windows10/comments/8h3eqo/windows_1803_white_bar_across_top_of_maximized/

See a video of the behavior in action: [http://recordit.co/3vTX9jpuVK](url)

**Comment by timfish** [comment]
This is a duplicate of #12971

**Comment by ahkohd** [comment]
> * Electron Version: 2.0.2
> * Operating System (Platform and Version): Windows 10 (17134)
> * Last known working Electron version: N/A
> 
> **Expected Behavior**
> When maximizing a frameless window, then unmaximizing and then maximizing again, there should not be a white bar atop the window.
> 
> **Actual behavior**
> The first maximize works. Removing maximize also works. Immediately maximizing again (without resizing the window) leaves a white bar several pixels high atop the browserwindow.
> 
> **To Reproduce**
> Use the latest Discord app (or various others like Spotify), follow the workflow above to reproduce this very easily.
> 
> Further discussion here: https://www.reddit.com/r/Windows10/comments/8h3eqo/windows_1803_white_bar_across_top_of_maximized/
> 
> See a video of the behavior in action: [http://recordit.co/3vTX9jpuVK](url)

**Comment by msavin** [comment]
The white line on top of the screen is transparent. If you can put a `#000000` line beneath it,  you can dim it quite a bit.

It's a bit tricky to get it to render, but adding the following to my custom title bar did the trick.

```css
border-top: 1px solid #000000
padding-top: 1px
margin-top: -1px
````

The white bar came down from `rgb(175,175,175)` to `rgb(151,151,151)`

Edit: this may also require the following inside the title bar:

```html
<div style="position: fixed; top: 0; left: 0; background: #000; height: 1px; width: 100%"></div>
```
